src/utils/alignments/index.ts

```diff
@@ -1,22 +1,44 @@
-export type HorizontalAlignment = 'leading' | 'center' | 'trailing';
-export type VerticalAlignment = 'top' | 'center' | 'bottom';
+export type TextAlignment = 'leading' | 'center' | 'trailing';
```

Comment:
I think the naming you had may fit better. SwiftUI uses `HorizontalAlignment`, `VerticalAlignment`, and `Alignment`, where  `Alignment` has all 9 locations, including `.topLeading` etc.

Not to be confused with `Edge` 😂

Reply:
okay, might have to switch that back